### ISSUE TITLE
order photos

### ISSUE THREAD

**Issue by chassain** [roramirez/MMM-ImagesPhotos#16]
Hello,

i am interresting to display photos in order and not ramdom.. If someone can writte this because i m beginner in java.

Thanks

**Comment by roramirez** [comment]
What type of order are you looking for?

Maybe set a configuration for not use a random mode can be help.

Do you want to do this yourself?

**Comment by chassain** [comment]
hello,

It's my first experience with javascript..i'm using MMM-Imageslideshow with randomorder (number). It's perfect for me..thanks !

**Comment by roramirez** [comment]
I've keep open this issue until to I'll adds this option.

**Comment by chassain** [comment]
Yes. Great...it is possible to integrate .pdf too ? Have a good day..(night in france )
